modules/reissue-internal-certificates-secured-clusters.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * configuration/reissue-internal-certificates.adoc
+:_module-type: PROCEDURE
+[id="reissue-internal-certificates-secured-cluster_{context}"]
+= Reissuing internal certificates for Secured Clusters using init bundles
+
+Secured clusters contain the Collector, Sensor, and Admission Control components. These components use a built-in server certificate for authentication when communicating with other {product-title} components.
+
+The RHACS portal shows an information banner when the Central certificate is about to expire.
+
+[NOTE]
+====
+The information banner only appears 15 days before the certificate expiry date.
+====
+
+.Prerequisites
+
+* To reissue certificates, you must have `write` permission for the `ServiceIdentity` resource.
+
+[IMPORTANT]
+====
+Store this bundle securely because it contains secrets.
+You can use the same bundle on multiple secured clusters.
+====
+
+.Procedure
+
+To generate an init bundle using the {product-title-short} portal:
+
+. Choose *Platform Configuration* -> *Clusters*.
+. Click *Manage Tokens*.
```

Comment:
Global comment for the use of the term "Click"
Per ISG: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#verbs-to-use

> If you write content that might be viewed both on mobile and non-mobile devices and using _click_ might cause confusion, you can use generic verbs such as select.

Reply:
Due to the use of "click" elsewhere in doc, and because it's desktop-only, I would prefer to use click to be consistent until we have a chance to further clean up the docs.